### ISSUE TITLE
Login Screen is Broken

### ISSUE THREAD

**Issue by nickde9999** [PFCKrutonium/Windows-10-Login-Background-Changer#168]
Hi 

Got a flashing screen after applying attached image. Broken PI as well
![windows ui logon pri new](https://cloud.githubusercontent.com/assets/13747892/9197196/9ed4ea7a-4032-11e5-9edd-6dcb829c131a.jpg)

![this-is-my-pc-43093-1920x1080](https://cloud.githubusercontent.com/assets/13747892/9197148/1e1264a8-4032-11e5-91ce-7f25df6252a5.jpg)

Replaced with .bak file and all is working again

Regards
Nick

**Comment by Englbach** [comment]
Me too and I had fix it. Please don't try this software.

**Comment by Toyz** [comment]
The software it's self works fine but if you use a highly detailed picture the Pri breaks there is nothing we can do to prevent it, I have personally tested the software on multiple machines and vms and it's never once made a bad PRI

I have recreated 90% of the issues by using a bad image size, or a highly detailed image there is nothing we can do about it so don't go around falsely accusing a software when you clearly don't know the issue

**Comment by Toyz** [comment]
As for the current picture about this issue I am not at my machine to test it but you could also try using my img2pri site I don't have the link off hand but if you go to the gitter chat someone can give you it

**Comment by nickde9999** [comment]
Well, then add maybe a warning to reduce the detail of the picture before attempting to change it.

**Comment by Toyz** [comment]
Ok and how am I to know how detailed the image is?

**Comment by frankjunior** [comment]
I've created this Github account to respond to Toyz. I do not believe this issue to be with the image that you use. I have experienced this issue as well. I've uploaded the program and the image to my NAS, then used it on my laptop. Even after running the program again after the Windows update that reverted the login pri, it is still holding stable. Using the same version of the program on my desktop, with the same background image, and the same Windows ISO to install Windows 10, all from the same location, it crashed my login as described. The laptop monitor and the desktop monitor run the same aspect ratio. If not the program itself, could it be the display drivers? For my laptop, running an old version of Intel's integrated graphics driver, while my desktop runs the current NVIDIA Geforce driver.
